projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-overview/table-overview.component.html

```diff
@@ -26,6 +26,22 @@ <h4 class="go-heading-4">loadingData</h4>
         </p>
       </div>
 
+      <div class="go-column go-column--100">
+        <h4 class="go-heading-4">renderBoxShadows</h4>
+        <p class="go-body-copy go-body-copy--no-margin">
+          This binding toggles box-shadows for the table. By default, it is set to <code class="code-block--inline">true</code>.
+          Typically used for embedding a table within an accordion.
+        </p>
+      </div>
+
+      <div class="go-column go-column--100">
+        <h4 class="go-heading-4">showTableActions</h4>
```

Comment:
I was confused for a moment because I know we have documentation on this lol. Good catch on adding it here too.